tests/e2e/paths/search/granules/granules.spec.js

```diff
@@ -0,0 +1,1197 @@
+          const request = route.request()
+          const body = request.postData()
+          expect(body).toBe('echo_collection_id=C1214470488-ASF&page_num=1&page_size=20&temporal=2000-01-20T00:00:00.000Z,2020-01-31T23:59:59.999Z,1,31')
+          route.fulfill({
+            body: JSON.stringify(recurringTemporalGranulesBody),
+            headers: {
+              ...commonGranulesHeaders,
+              'access-control-expose-headers': 'cmr-hits',
+              'cmr-hits': cmrHits
+            }
+          })
+        })
+
+        await page.route('**/search/granules/timeline', (route) => {
+          const request = route.request()
+          const body = request.postData()
+          if (body) {
+            expect(body).toBe('end_date=2023-12-01T00:00:00.000Z&interval=day&start_date=2018-12-01T00:00:00.000Z&concept_id[]=C1214470488-ASF')
+          }
+
+          route.fulfill({
+            json: temporalTimelineBody,
+            headers: temporalTimelineHeaders
+          })
+        })
+
+        await page.route('**/api', (route) => {
+          const request = route.request()
+          const body = JSON.stringify(request.postData())
+          const expectedBody = JSON.stringify(graphQlGetCollection(conceptId))
+
+          const requestBody = JSON.parse(body)
+          const expectedRequestBody = JSON.parse(expectedBody)
+
+          expect(requestBody).toEqual(expectedRequestBody)
+          route.fulfill({
+            body: JSON.stringify(temporalGraphQlBody),
+            headers: temporalGraphQlHeaders
+          })
+        })
+
+        await page.goto('/search/granules?p=C1214470488-ASF&pg[0][qt]=2000-01-20T00:00:00.000Z,2020-01-31T23:59:59.999Z,1,31')
+
+        // Ensure the correct number of results were loaded
+        await testResultsSize(page, cmrHits)
+
+        // Readable granule name input is empty
+        await expect(page.getByTestId('granule-filters__readable-granule-name')).toHaveValue('')
+
+        // Temporal is populated
+        await expect(page.locator('#granule-filters__temporal-selection__temporal-form__start-date')).toHaveValue('01-20 00:00:00')
+        await expect(page.locator('#granule-filters__temporal-selection__temporal-form__end-date')).toHaveValue('01-31 23:59:59')
+        await expect(page.locator('#granule-filters__temporal-selection__recurring')).toBeChecked()
+        await expect(page.locator('.temporal-selection__range-label')).toHaveText('2000 - 2020')
+      })
+    })
+  })
+
+  test.describe('When the path is loaded with browse only set to true', () => {
+    test('loads with the browse only checkbox checked', async ({ page }) => {
+      const conceptId = 'C1214470488-ASF'
+      const cmrHits = 0
+
+      await page.route('**/search/granules.json', (route) => {
+        const request = route.request()
+        const body = request.postData()
+        expect(body).toBe('browse_only=true&echo_collection_id=C1214470488-ASF&page_num=1&page_size=20')
```

Comment:
We generally have new lines to separate blocks of variables, expect blocks, and function calls. There are lots of occurrences of this in this file. Make sure all the spacing is consistent please :)

Reply:
Fixed